qdrant_client/conversions/conversion.py

```
@@ -4280,6 +4282,8 @@ def convert_recommend_strategy(cls, model: rest.RecommendStrategy) -> grpc.Recom
             return grpc.RecommendStrategy.AverageVector
         elif model == rest.RecommendStrategy.BEST_SCORE:
             return grpc.RecommendStrategy.BestScore
+        elif model == reset.RecommendStrategy.SUM_SCORES:
```

Ruff (0.8.2)

Undefined name `reset`

(F821)